code/examples/official-storybook/stories/addon-actions.stories.js

```diff
@@ -1,196 +0,0 @@
-/* eslint-disable react/prop-types */
-import global from 'global';
-import React, { Fragment } from 'react';
-import { action, actions, configureActions } from '@storybook/addon-actions';
-import { Form } from '@storybook/components';
-
-const { window: globalWindow, File } = global;
-const { Button } = Form;
-
-export default {
-  title: 'Addons/Actions',
-  parameters: {
-    options: {
-      selectedPanel: 'storybook/actions/panel',
-    },
-  },
-};
-
-export const ArgTypesExample = ({ onClick, onFocus }) => (
-  <Button {...{ onClick, onFocus }}>Hello World</Button>
-);
-
-ArgTypesExample.argTypes = {
-  onClick: { action: 'clicked!' },
-  onFocus: { action: true },
-};
-
-export const ArgTypesRegexExample = (args, context) => {
-  const { someFunction, onClick, onFocus } = args;
-  return (
-    <Button onMouseOver={someFunction} {...{ onClick, onFocus }}>
-      Hello World
-    </Button>
-  );
-};
-
-ArgTypesRegexExample.parameters = { actions: { argTypesRegex: '^on.*' } };
-ArgTypesRegexExample.argTypes = { someFunction: {}, onClick: {}, onFocus: {} };
```

Comment:
Are we missing these argType-driven examples?

Reply:
added!